benchmark/sample_genes.py

```diff
@@ -74,6 +76,72 @@ def sample_genes(ortho_groups_fp,
     return genes
 
 
+def write_genes(genes,
+                input_faa_dir,
+                output_fa_dir,
+                output_genes_fp):
+    """ Write protein sequences of selected gene families to external files
+
+    Parameters
+    ----------
+    genes : dict of dict of set of str
+        { ogid : { taxon : set(protein(s)) }
```

Comment:
I have some experience with scikit-bio's Sequence object, but I couldn't think of a way to take advantage of it, otherwise I already did (see `io` below). Do you have a clue for it?
Maybe **pandas** instead of **scikit-bio** is the right package for this case. Still I couldn't think of a solution with pandas. Therefore I did the most native way.